modules/local/locidex/merge/main.nf

```diff
@@ -7,8 +7,9 @@ process LOCIDEX_MERGE {
     label 'process_medium'
 
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-    'docker.io/mwells14/locidex:0.2.3' :
-    'docker.io/mwells14/locidex:0.2.3' }"
+    "docker.io/mwells14/locidex:0.2.3" :
+    task.ext.override_configured_container_registry != false ? 'docker.io/mwells14/locidex:0.2.3' :
+    'mwells14/locidex:0.2.3' }" 
```

Comment:
@apetkau A test of my comprehension of what we discussed in the`iridanextexample` [(here)](https://github.com/phac-nml/iridanextexample/pull/31): Since the singularity container is still a docker hub (converted to singularity), and because the override won't use the mwells14 dockerhub container we can write it as:
```bash
    container "${ task.ext.override_configured_container_registry != false ?
    'docker.io/mwells14/locidex:0.2.3' :
    'locidex:0.2.3' }"
  ```
  P.S It looks like Nextflow 24.04 will be improving the docker->singularity process (see [here](https://nextflow.io/blog/2024/nextflow-2404-highlights.html#oci-auto-pull-mode-for-singularity-and-apptainer))

Reply:
@sgsutcliffe yes, that's correct. We could remove checks for e.g., `singularity_pull_docker_container` since they aren't really used in this case. Though I would expect locidex to eventually be released on quay.io/biocontainers (and so be cached in the depot.galaxyproject.org site) in a later update.

So I think either option of leaving this as-is or changing it are both okay.

Thanks so much for the link to updates on the docker -> singularity process 😄

Reply:
Although the new approach offers a cleaner look, I am going to maintain the current setup for consistency with `gasclustering` and other pipelines. This will also streamline the process when `locidex` is added to Quay, as we’ll only need to replace the image without reintroducing the container directive check.